growth/too/tool.py

```diff
@@ -95,13 +95,15 @@ def create(sample):
         models.db.session.merge(models.User(name='fritz'))
         models.db.session.commit()
 
-        for filename in ['tests/data/GRB180116A_Fermi_GBM_Alert.xml',
-                         'tests/data/GRB180116A_Fermi_GBM_Flt_Pos.xml',
-                         'tests/data/GRB180116A_Fermi_GBM_Gnd_Pos.xml',
-                         'tests/data/GRB180116A_Fermi_GBM_Fin_Pos.xml',
-                         'tests/data/MS181101ab-1-Preliminary.xml',
-                         'tests/data/MS181101ab-4-Retraction.xml',
-                         'tests/data/AMON_151115.xml']:
+        for filename in ['tests/data/S190425z.xml',
+                         'tests/data/S190425z_lalinference.xml']:
+#                         'tests/data/GRB180116A_Fermi_GBM_Alert.xml',
```

Comment:
Uh oh. We don't want to disable all of these unit tests.

Reply:
Yes, sorry, I had accidentally committed a change I was using for testing the code. This should be fixed.